packages/common/src/index.ts

```diff
@@ -959,8 +989,24 @@ export default class Common extends EventEmitter {
    * ethash: -
    * clique: period, epoch
    * aura: -
+   * casper: -
+   *
+   * Note: This value can update along a hardfork.
    */
   consensusConfig(): any {
+    const hardfork = this.hardfork()
+
+    let value
+    for (const hfChanges of HARDFORK_CHANGES) {
+      if (hfChanges[1].hasOwnProperty('consensus')) { // eslint-disable-line
```

Comment:
@holgerd77 I thought you were a fan of the `in` operator :) these should be equivalent right?
```suggestion
      if ('consensus' in hfChanges[1]) {
```

Reply:
👍